Sources/JavaKitExample/JavaKitExample.swift

```diff
@@ -103,7 +103,7 @@ struct HelloSwift {
 }
 
 extension JavaClass<HelloSwift> {
-  @JavaField
+  @JavaStaticField
   var initialValue: Double
```

Comment:
should this be a static var?

Reply:
It's an instance in JavaClass similar to how static methods are instance methods on their Java class. I was thinking of how to get this be a static var, but everything needs the JNIEnvironment (from my understanding), which is an instance var on all Java structs

Reply:
Right. Java static methods/vars become non-static members on the appropriately-specialized `JavaClass` instance.

Reply:
For bonus points, one could have the `@JavaStaticField` macro detect the `static` and produce an error diagnosing the problem, but it's not really necessary. Folks shouldn't have to write these macro uses themselves.

Reply:
Hmmm do we have access to the surrounding context of a macro? I know you did some work there, but I lost where that ended up.

Reply:
Yeah, the [`lexicalContext`](https://github.com/swiftlang/swift-syntax/blob/main/Sources/SwiftSyntaxMacros/MacroExpansionContext.swift#L57-L72) of the macro captures this information. To be very clear, I don't think this diagnostic is something that should block your PR from being merged. I just wanted to note that we can implement these kinds of checks  in the macro implementation if we want.